policies/recipes/steps.go

```diff
@@ -131,27 +132,100 @@ func StepFileExec(step *osconfigpb.SoftwareRecipe_Step_FileExec, artifacts map[s
 
 // StepScriptRun builds the command for a ScriptRun step
 func StepScriptRun(step *osconfigpb.SoftwareRecipe_Step_ScriptRun, artifacts map[string]string, runEnvs []string, stepDir string) error {
```

Comment:
I don't have access to a physical windows machine so I couldn't actually run any of these code paths, what is the best way to actually run this code without one? My current plan is just to spin up a windows VM on GCE and run it there.